font/src/lib.rs

```diff
@@ -267,3 +278,11 @@ pub trait Rasterize {
     /// Update the Rasterizer's DPI factor
     fn update_dpr(&mut self, device_pixel_ratio: f32);
 }
+
+// Only implemented for the FreeType rasterizer so far.
```

Comment:
I think leaving a comment here will make it easy to just forget about it even after implementation. Missing features are better documented in github issues, where we don't have to go crawl through code to find remaining TODOs (especially since this isn't even tagged as TODO).

So I'd just remove this and we can then create a new issue about ligature support on Windows and macOS.

Reply:
I've reworded the comment to hopefully better capture what it does. 

At a larger scope, I actually think this trait might go away entirely as font ligature support is added to windows and mac osx. Both of their APIs appear (from a quick glance) to rasterize batches of glyphs at a time and expose shaping through that. Freetype is the odd man out in that regard, so I'd imagine the Rasterizer trait would change to rasterize batches of glyphs instead of one glyph at a time and then Harfbuzz would entirely be an implementation detail of the freetype rasterizer.

However until taking a further look at the directwrite and core text apis I don't want to take a stab at modifying rasterizer so for now this is an okay approach.

Reply:
My problem with this comment here is that it doesn't serve much of a purpose and it requires maintenance in the future when the trait is implemented for another platform. Having ta change this to say it's implemented for FreeType and CoreText would be a waste for example. And since this is harfbuzz specific at the moment anyways, there's not really a good way to reuse it on other platforms as far as I'm aware.

So in general this specific comment just doesn't help much imho. If I want to see where the trait is used, I can just look at the references in the code.